### ISSUE TITLE
bug in displaying numpy image while debugging

### ISSUE THREAD

**Issue by morhadar** [elazarcoh/simply-view-image-for-python-debugging#30]
Hi,
First - great extension! thank you very much

I ran into a bug when referring to the bellow code:
```
from PIL import Image
import numpy as np

image = Image.open('data/sandbox/00599.jpg')
print(image.getpixel((0,0)))
image_np = np.array(image)
print(image_np[0,0,:]) #if stops here with debugger and presents image I get undesired prints to std
image_pil = Image.fromarray(image_np)
print(image_pil.getpixel((0,0)))
```
If I run it once, without any breaking point, it prints the expected result to std:
(docscanner-env) mhadar@mhadar-lx:~/projects/doc-scanner$  cd /home/mhadar/projects/doc-scanner ; /usr/bin/env /home/mhadar/anaconda3/envs/docscanner-env/bin/python /home/mhadar/.vscode-server/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/adapter/../../debugpy/launcher 34741 -- /home/mhadar/projects/doc-scanner/tmp.py 
**(14, 14, 6)
[14 14  6]
(14, 14, 6)**

But if I run this code with a breaking point where the comment is and present 'image_np' from the 'IMAGE WATCH' I get the undesired prints to std:
(docscanner-env) mhadar@mhadar-lx:~/projects/doc-scanner$  cd /home/mhadar/projects/doc-scanner ; /usr/bin/env /home/mhadar/anaconda3/envs/docscanner-env/bin/python /home/mhadar/.vscode-server/extensions/ms-python.python-2022.14.0/pythonFiles/lib/python/debugpy/adapter/../../debugpy/launcher 38931 -- /home/mhadar/projects/doc-scanner/tmp.py 
**(14, 14, 6)
[83 83 35]
(83, 83, 35)**

and also the image is not presented correctly as you can see here:
![image](https://user-images.githubusercontent.com/17726534/189059628-b7379393-a035-4a2b-adf0-49d71f904282.png)

I think that even if there is a bug in the presentation the extension should never change the original object (maybe the solution should be doing a hard copy for the image before processing it for presentation?)
And of course, there is a bug in the presentation of NumPy image (I think this only happens if I work with the PIL package in parallel).

Again, greater extension! love working with it!

**Comment by morhadar** [comment]
Thanks for the fast response!
This is the image.
![00599](https://user-images.githubusercontent.com/17726534/189068954-27216c43-9df1-4d21-ac0c-f20d1130df9b.jpg)

I now suspect that It may be a collision with a different presenting images extension. let me check this out.

**Comment by morhadar** [comment]
I disabled the other extension and the bug keeps occurring.
Now I see it only happens when presenting images from the 'IMAGE WATCH' menu. If I display the image using the bulb of the code action it works as expected

**Comment by elazarcoh** [comment]
This should be now fixed in version 2.3.0.
The problem was it was displayed as a tensor (which had a bug). Now you can display it either as a tensor or as an image.
